src/Microsoft.AspNet.Localization/ApplicationBuilderExtensions.cs

```diff
@@ -63,12 +33,7 @@ public static class ApplicationBuilderExtensions
                 throw new ArgumentNullException(nameof(options));
             }
 
-            if (defaultRequestCulture == null)
```

Comment:
We checked that we didn't set the DefaultRequestCulture to null, but are we checking that it has ever been set anywhere or did I miss it?